modal_proto/api.proto

```diff
@@ -1948,6 +1968,7 @@ message RuntimeOutputBatch {
 message RuntimeInputMessage {
   bytes message = 1;
   uint64 message_index = 2;
+  bool is_eof = 3;
```

Comment:
I considered moving `message` and `is_eof` into a `oneof`, because `message` would be ignored if `is_eof` is true. However, that complicates the Rust code quite a bit so keeping it as is. Any strong opinions here?

Reply:
`is_eof` -> `eof`